packages/sdk-core/src/common/jsonSize.ts

```diff
@@ -0,0 +1,131 @@
+function stringifiedSize<T>(value: T): number {
+    return JSON.stringify(value).length;
+}
+
+function toStringSize<T extends { toString(): string }>(value: T): number {
+    return value.toString().length;
+}
+
+const stringSize = (value: string) => stringifiedSize(value);
+const numberSize = toStringSize<number>;
+const bigintSize = toStringSize<bigint>;
+const symbolSize = 0;
+const functionSize = 0;
+const booleanSize = (value: boolean) => (value ? 4 : 5);
+const undefinedSize = 0;
+const nullSize = 'null'.length;
+
+const arraySize = (array: unknown[], replacer?: (this: unknown, key: string, value: unknown) => unknown): number => {
+    const bracketLength = 2;
+    const commaLength = array.length - 1;
+    let elementsLength = 0;
+    for (let i = 0; i < array.length; i++) {
+        const element = array[i];
+        switch (typeof element) {
```

Comment:
yes, because the behavior differs a bit:
![image](https://github.com/backtrace-labs/backtrace-javascript/assets/9085864/142ea59c-5eff-4e06-976f-ea32927fbdc0)